pkg/up/util.go

```diff
@@ -116,13 +116,14 @@ func inspectImageRawParseHealthcheck(inspectRaw []byte) (*config.Healthcheck, er
 
 // resolveLocalImageID resolves an image ID against a cached list (like the one output by the command "docker images").
 // ref is assumed not to be a partial image ID.
+// nolint
```

Comment:
Add TODO issue ref or justification of nolint

Reply:
done